packages/markup-frontstage/README.md

```diff
@@ -0,0 +1,52 @@
+# @bentley\markup-frontstage-react
```

Comment:
should be @bentley/markup-frontstage-react

Reply:
thanks, corrected please :)